source/sync/configure/sync-settings.txt

```diff
@@ -372,8 +499,29 @@ Sync Config Object
    * - | ``queryable_fields_names``
        | Array<String>
      
-     - The :ref:`names of the fields <queryable-fields>` that your client 
-       application can query to determine which data to synchronize.
+     - The :ref:`names of the global queryable fields <queryable-fields>` 
+       that your client application can query to determine which data to 
+       synchronize. These fields are queryable across all synced collections.
+
+   * - | ``indexed_queryable_fields_names``
+       | Array<String>
+     
+     - The :ref:`name of the indexed queryable field <fs-indexed-queryable-fields>` 
```

Comment:
Also probably worth noting here that the entry must also occur in `queryable_fields_names` since it needs to be a global queryable field